src/Util/ClassSourceManipulator.php

```diff
@@ -178,6 +180,40 @@ public function addInterface(string $interfaceName)
         $this->updateSourceCodeFromNewStmts();
     }
 
+    public function addTrait($classTraitName)
+    {
+        $this->addUseStatementIfNecessary($classTraitName);
```

Comment:
I think this should be:

```suggestion
        $name = $this->addUseStatementIfNecessary($classTraitName);
```

The `addUseStatementIfNecessary` returns the "short name" that is now available to be used through the rest of the class. Then, the next 2 lines should not be needed.

This catches edge-case situations where the `use` statement is already on the class, but maybe is aliased to something.

Reply:
Okay, done.